From f81c7af25d1fe00b2729f27967358f8c494bc957 Mon Sep 17 00:00:00 2001
From: Kei Okada <kei.okada@gmail.com>
Date: Fri, 19 Dec 2014 17:37:33 +0900
Subject: [PATCH 1/2] do not ask for users to generate doc

---
 setup.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 3228d2d..0e447e4 100644
--- a/setup.py
+++ b/setup.py
@@ -43,7 +43,8 @@ def get_files(dir, ext=None):
 
 if sys.platform != 'win32':
     cwd = os.path.join(os.getcwd(), 'doc')
-    s = raw_input('Generate documentation? ')
+    #s = raw_input('Generate documentation? ')
+    s = 'y'
     if s.lower() == 'y' or s.lower() == 'YES':
         print 'Generating documentation'
         p = subprocess.Popen(['./make_docs', 'man', 'html', 'pdf', '-v'],
-- 
1.9.1

